cmd/ocm-backplane/cloud/credentials.go

```diff
@@ -119,22 +119,32 @@ func runCredentials(cmd *cobra.Command, argv []string) error {
 	if isolatedBackplane {
 		targetCredentials, err := getIsolatedCredentials(clusterID)
 		if err != nil {
-			return fmt.Errorf("failed to get cloud credentials for cluster %v: %w", clusterID, err)
-		}
-
-		bpCreds := &bpCredentials.AWSCredentialsResponse{
-			AccessKeyID:     targetCredentials.AccessKeyID,
-			SecretAccessKey: targetCredentials.SecretAccessKey,
-			SessionToken:    targetCredentials.SessionToken,
-			Expiration:      targetCredentials.Expires.String(),
-		}
-		if region, ok := cluster.GetRegion(); ok {
-			bpCreds.Region = region.ID()
-		}
-
-		output, err = renderCloudCredentials(credentialArgs.output, bpCreds)
-		if err != nil {
-			return fmt.Errorf("failed to render credentials: %w", err)
+			// itn-2023-00143 handle case where customer's org is on the isolated flow,
+			// but they have not yet migrated their account roles
+			fmt.Println("Cluster's org is using new flow but cluster has not migrated to new account roles. Trying old flow...")
+			credsResp, err := getCloudCredential(bpURL, clusterID)
+			if err != nil {
+				return fmt.Errorf("failed to get cloud credentials for cluster %v: %w", clusterID, err)
+			}
+			output, err = renderCredentials(credsResp.JSON200.Credentials, credsResp.JSON200.Region, cloudProvider)
+			if err != nil {
+				return err
+			}
+		} else {
+			bpCreds := &bpCredentials.AWSCredentialsResponse{
+				AccessKeyID:     targetCredentials.AccessKeyID,
+				SecretAccessKey: targetCredentials.SecretAccessKey,
+				SessionToken:    targetCredentials.SessionToken,
+				Expiration:      targetCredentials.Expires.String(),
+			}
```

Comment:
My understanding is that the old flow returns a region, something like this
```suggestion
			bpCreds := &bpCredentials.AWSCredentialsResponse{
				AccessKeyID:     targetCredentials.AccessKeyID,
				Region          string `json:"Region,omitempty" yaml:"Region,omitempty"`
				SecretAccessKey: targetCredentials.SecretAccessKey,
				SessionToken:    targetCredentials.SessionToken,
				Expiration:      targetCredentials.Expires.String(),
			}
```